_posts/2024-03-08-international-womens-day-2024.md

```diff
@@ -0,0 +1,63 @@
+---
+title: International Women's Day
+authors:
+- name: Yann Eves
+  link: https://yannev.es
+collection: news
+layout: news.html
+---
+
+Today is [International Women's Day][international-womens-day]. We celebrated Async's 14th Birthday yesterday and our community - and the tech industry as a whole - has become more diverse in those years. An inclusive, welcoming community benefits _everyone_ and we still have much to do, both globally and locally.
+
+The theme for International Women's Day 2024 is **inspire inclusion**. So we want to share some work we've been doing behind the scenes that inspires inclusion in our community, and our upcoming plans to continue beating that drum.
+
+### Challenges
+
+In our journey to promote diversity and inclusion, we've encountered several hurdles:
+
+- **Lack of talks from women in tech**: Curating speakers is the hardest part of organising Async, we rely a lot on speakers volunteering from within our community (thank you!). Despite our efforts to host talks from women in tech, many of our past speakers are on an indefinite hiatus, posing a significant challenge as we have to reach out further afield.
+
+- **Big audience**: The increased attendance at Async events has made it intimidating for new speakers to step forward, hindering our efforts to support and nurture diverse voices.
+
+- **Venue accessibility**: Our past venues, while generously provided for free, posed accessibility challenges with poor lighting and off-street location, making it feel less safe to travel to on your own.
+
+### Our response
+
+To address these challenges and inspire inclusion, we've taken proactive steps:
+
+- **Establishment of a speaker fund**: We're creating a fund to cover transportation and accommodation costs for speakers, particularly those from underrepresented backgrounds, to ensure that speaking opportunities are accessible to all.
+
+- **Search for a more accessible venue**: We started searching for a venue that prioritises accessibility, including well-lit spaces with clear signage and convenient street access.
+
+- **Improving event visibility**: We're investing in signage and branded clothing to make organisers more identifiable at events, fostering a welcoming environment for attendees.
+
+- **Language review**: We're reviewing our event promotions to ensure they're inclusive and free from accidental discrimination.
+
+- **Supporting novice speakers**: We're reintroducing our 'twofer' event format to provide opportunities for novice speakers to share their voices and dedicating more volunteer time to support them.
```

Comment:
Is it worth explaining what a twofer is?

Reply:
"two speakers share the stage"